usagov/usagov-2021
# USAGOV-1989: Page intro as fallback for meta description on wizards

https://cm-jira.usa.gov/browse/USAGOV-1989

## Description
When the mate_description is empty, we use the intro field instead for the meta-tag output.

## Type of Changes


## Testing Instructions
* Edit a term like https://localhost/taxonomy/term/67/edit and empty its Meta-Description field. 
* Make sure there is something in the Intro field.
* Save, and load the page on the front end. Inspect the source-code of the front-end page (view page source)
* Look for the `<meta` tag that contains the "description" of the page. The value for the meta-tag should be the contents of the Intro field now 

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
